webapp/src/pages/reviewDetails/index.js

```diff
@@ -0,0 +1,171 @@
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import ReactTable from "react-table";
+import { withRouter } from "react-router";
+import "react-table/react-table.css";
+
+import { reviewService } from "../../services/reviews";
+import { withTranslation } from 'react-i18next'
+
+class ReviewDetailsPage extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            error: null,
+            infoColumns: [],
+            isLoading: true,
+            reviewDetails: [],
+        }
+        this.identifierCell = this.identifierCell.bind(this);
+        this.responseIDCell = this.responseIDCell.bind(this);
```

Comment:
I believe this isn't necessary when using arrow/lambda functions to define the methods, because `this` is bound to the defining scope rather than the calling object with arrow functions.

Reply:
Thanks for catching this. I originally had them as regular ol' functions and then changed them to arrow functions in order to have consistency with the other components in the app.